### DISCUSSION TITLE
Mobile version

### DISCUSSION THREAD

**Discussion by knoxydev** [FacilMap/facilmap#234] [Ideas]
What about to create mobile version ? for example for ios/android ?

**Comment by cdauth** [comment]
I'm not a regular user of smartphones myself, but I tried my best to make the UI usable on small touch screens. You can find information how to install FacilMap as an app (a progressive web app to be precise) [in the docs](https://docs.facilmap.org/users/app/). Is there anything specific that you are missing?

**Reply by knoxydev** [reply]
thanks for the tip. I have installed a web application

**Comment by SecularSteve** [comment]
I have a question relating to this.

If someone were to host an app edition of this on F-Droid, or AltStore or something, you wouldn't mind it?

**Reply by cdauth** [reply]
I think if someone wants to make an app, the better approach would be to contribute the code to the existing code base and I take care of the publishing. But I'm still wondering whether having a “real” mobile app would have any advantages over a progressive web app?

**Reply by SecularSteve** [reply]
Think of it this way.

Having a more simple anti-thesis to an app like OsmAnd would mean a lot in a piece of the App Market rarely taken advantage of.

Simple, Elegant, Usable, Resource-Light and above all else Completely Free.

And besides, I never knew anyone who'd open a Maps Program inside a browser on their phone for the explicit intent of navigation.
